proposals/001_Alternative Schema Proposal.md

```diff
@@ -53,7 +53,7 @@ Examples of the use of the Alternative Schema capability is added to the OpenAPI
 
 ### Alternative Schema Registry
 
-Values used to populate the Alternative Schema Object are required to come from the Alternative Schema Registry. The preliminary Alternative Schema Registry is located at https://spec.openapis.org/registries/alternative-schema[https://spec.openapis.org/registries/alternative-schema].
+Values used to populate the Alternative Schema Object are required to come from the Alternative Schema Registry. The preliminary Alternative Schema Registry is located at [https://spec.openapis.org/registries/alternative-schema](https://spec.openapis.org/registries/alternative-schema).
```

Comment:
```suggestion
Values used to populate the Alternative Schema Object are required to come from the Alternative Schema Registry. The preliminary Alternative Schema Registry is located at <https://spec.openapis.org/registries/alternative-schema>.
```

if URL and link name are the same you can use `<>` in markdown :)

Reply:
You can actually use a URL bare in github markdown, and it will be convered, however others may download the file to review it in markdown locally.

Reply:
`<>` marks it as URL explicitly, without annotation github will guess what is an URL and what is not. https://github.github.com/gfm/#autolinks

Reply:
I'm assuming to keep this as valid markdown and not just valid GFM.
If I check out this repo locally and want to test changes, I need a local markdown viewer.

Reply:
sorry, for not being clear about this, `<>` are not a github feature, it is in original markdown also: https://daringfireball.net/projects/markdown/syntax#autolink

Reply:
TIL. Thanks!